README.md

```diff
@@ -40,28 +40,29 @@
 - JSX
 - CSS
 - React JS
+- Redux
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
--[x] **Rockets**
--[x] **Dragons**
--[x] **Missions**
--[x] **Profile**
+-[x] **Serach coin**
+-[x] **Crypto coins list**
+-[x] **Crypto coins details**
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
-## 🚀 Live Demo <a name="live-demo"></a>(https://spacex-3bz6.onrender.com/)
+## 🚀 Live Demo <a name="live-demo"></a>(Coming soon)
 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- Loom Walkthrough -->
 ## LOOM PROJECT PRESENTATION <a name="loom-demo"></a>
 
-- [Loom Walkthrough Link](coming soon)
+- [Loom Walkthrough Link](https://www.loom.com/share/55e4214003fe47449424e4eb5f6b408b)
```

Comment:
- [ ] As requested by the previous code reviewer you must deploy your app to approve your project, additionally, it is a mobile version so please 🙏🏽 show the correct size in the video, and kindly show a piece of code that you feel proud of

Reply:
@franclobo How I can deploy when the main branch is empty? I thought that the deployment is done from the main branch. Should I record another video?